policyengine_us/variables/gov/states/wv/tax/income/credits/wv_low_income_family_tax_credit.py

```diff
@@ -0,0 +1,50 @@
+from policyengine_us.model_api import *
+
+
+class wv_low_income_family_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "West Virginia low-income family tax credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "wv_low_income_family_tax_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        filing_statuses = filing_status.possible_values
+        eligible = tax_unit("wv_low_income_family_tax_credit_eligible", period)
```

Comment:
```suggestion
```

Reply:
Yes we did, but we didn't specify if this condition is true then do the calculation, if it is false, then 0.

Reply:
if we use it in the defined_for - the whole calculation will go to 0 regardless, we dont need to specify it again in the formula

Reply:
Got it.